flepimop/main_scripts/create_seeding_added.R

```diff
@@ -91,18 +90,18 @@ if (is.null(config$seeding$ratio_incidC)) {
 # ~ Load ground truth data ------------------------------------------------
 #  -- this is already saved from running the `build_[X]_data.R` script at the model setup stage.
 #
-data_path <- config$inference$gt_data_path
-if (is.null(data_path)) {
-    data_path <- config$seeding$casedata_file
-    if (is.null(data_path)) {
+gt_data_path <- config$inference$gt_data_path
+if (is.null(gt_data_path)) {
+    gt_data_path <- config$seeding$casedata_file
+    if (is.null(gt_data_path)) {
         stop(paste(
             "Please provide a ground truth file",
             " as inference::gt_data_path or seeding::casedata_file"
         ))
     }
 }
-cases_deaths <- readr::read_csv(data_path)
-print(paste("Successfully loaded data from ", data_path, "for seeding."))
+cases_deaths <- readr::read_csv(gt_data_path)
+print(paste("Successfully loaded data from ", gt_data_path, "for seeding."))
```

Comment:
something weird w/ spaces here - either there's auto space, and the first string is wrong, or there isn't, and the last string is wrong.